nautobot/dcim/tests/test_models.py

```diff
@@ -1264,6 +1264,27 @@ def test_child_devices_are_not_saved_when_unnecessary(self):
         self.assertNotEqual(child_mtime_after_parent_rack_update_save, child_mtime_after_parent_site_update_save)
 
 
+class DeviceBayTestCase(ModelTestCases.BaseModelTestCase):
+    model = DeviceBay
+
+    def setUp(self):
+        devices = Device.objects.all()[:3]
+        devices[1].device_type.subdevice_role = SubdeviceRoleChoices.ROLE_CHILD
+        devices[1].device_type.validated_save()
```

Comment:
Is this "safe" to do, or would we be better to create a dedicated device and device-type of role `child`? Naively I would expect that changing the device-type like this might invalidate any existing devices that are serving as `parent` devices?